sktime/classification/compose/_ensemble.py

```diff
@@ -350,15 +347,18 @@ def predict(self, X):
     def predict_log_proba(self, X):
         """
         Predict class log-probabilities for X.
```

Comment:
"""Predict class log-probabilities for X.

Reply:
Done 👍